sql/core/src/main/scala/org/apache/spark/sql/execution/command/AnalyzeColumnCommand.scala

```diff
@@ -62,7 +62,7 @@ case class AnalyzeColumnCommand(
       val statistics = Statistics(
         sizeInBytes = newTotalSize,
         rowCount = Some(rowCount),
-        colStats = columnStats ++ catalogTable.stats.map(_.colStats).getOrElse(Map()))
+        colStats = catalogTable.stats.map(_.colStats).getOrElse(Map()) ++ columnStats)
```

Comment:
Could you leave a code comment here to emphasize it?  I am just afraid this might be modified without notice. Newly computed stats should override the existing stats.